docs/gen-docs/modulectl_create.md

```diff
@@ -13,73 +13,61 @@ Use this command to create a Kyma module, bundle it as an OCI artifact, and push
-Such projects require providing an explicit path to the module's project directory using the "--path" flag or invoking the command from within that directory.
+### Configuration
 
-### Simple mode configuration
-
-To configure the simple mode, provide the `--module-config-file` flag with a config file path.
+Provide the `--module-config-file` flag with a config file path.
 The module config file is a YAML file used to configure the following attributes for the module:
 
 ```yaml
 - name:             a string, required, the name of the module
 - version:          a string, required, the version of the module
 - channel:          a string, required, channel that should be used in the ModuleTemplate CR
+- manifest:         a string, required, reference to the manifest, must be a relative file name or URL
+
+- defaultCR:        a string, optional, reference to a YAML file containing the default CR for the module, must be a relative file name or URL
 - mandatory:        a boolean, optional, default=false, indicates whether the module is mandatory to be installed on all clusters
-- manifest:         a string, required, reference to the manifest, must be a relative file name
-- defaultCR:        a string, optional, reference to a YAML file containing the default CR for the module, must be a relative file name
 - resourceName:     a string, optional, default={NAME}-{CHANNEL}, the name for the ModuleTemplate CR that will be created
-- security:         a string, optional, name of the security scanners config file
 - internal:         a boolean, optional, default=false, determines whether the ModuleTemplate CR should have the internal flag or not
 - beta:             a boolean, optional, default=false, determines whether the ModuleTemplate CR should have the beta flag or not
+- security:         a string, optional, name of the security scanners config file
 - labels:           a map with string keys and values, optional, additional labels for the generated ModuleTemplate CR
 - annotations:      a map with string keys and values, optional, additional annotations for the generated ModuleTemplate CR
 ```
 
-The **manifest** and **defaultCR** paths are resolved against the module's directory, as configured with the "--path" flag.
 The **manifest** file contains all the module's resources in a single, multi-document YAML file. These resources will be created in the Kyma cluster when the module is activated.
-The **defaultCR** file contains a default custom resource for the module that will be installed along with the module.
-The Default CR is additionally schema-validated against the Custom Resource Definition. The CRD used for the validation must exist in the set of the module's resources.
+The **defaultCR** file contains a default custom resource for the module that will be installed along with the module. It will be additionally schema-validated against the Custom Resource Definition.
```

Comment:
```suggestion
The **defaultCR** file contains a default custom resource for the module that will be installed along with the module. It will be additionally schema-validated against the Custom Resource Definition.
```
```suggestion
The **defaultCR** file contains a default custom resource for the module that is installed along with the module. It is additionally schema-validated against the Custom Resource Definition.
```